content/docs/cucumber/state.md

```diff
@@ -238,6 +238,189 @@ compile group: 'io.cucumber', name: 'cucumber-needle', version: '{{% version "cu
 
 There is no documentation yet, but the code is on [GitHub](https://github.com/cucumber/cucumber-jvm/tree/master/needle).
 
+# Details of using DI
+
+(since Cucumber 5)
+
+When using a DI framework all your step definitions, hooks, transformers, etc. will be created by the frameworks instance injector.
+
+## The need for a custom injector
+
+Cucumber example tests are typically small and have no dependencies.
+In real life, though, tests often need access to application specific object instances
+which also need to be supplied by the injector.
+These instances need to be made available to your step definitions so that actions
+can be applied on them and delivered results can be tested.
+
+The reason using Cucumber with a DI framework typically originates from the fact that the tested application also uses
+the same framework. So we need to configure a custom injector to be used with Cucumber.
+This injectors ties tests and application instances together.
```

Comment:
```suggestion
This injector ties tests and application instances together.
```

I guess that we are talking about 1 injector.

Reply:
You are right. Thx. Fixed.